pkg/systemd/generate/containers_test.go

```diff
@@ -674,7 +797,13 @@ Restart=on-failure
 StartLimitBurst=42
 TimeoutStopSec=70
 ExecStartPre=/bin/rm -f %t/%n.ctr-id
-ExecStart=/usr/bin/podman run --name=container-foo-%i --cidfile=%t/%n.ctr-id --cgroups=no-conmon --rm --sdnotify=conmon -d awesome-image:latest
+ExecStart=/usr/bin/podman run \
+	--name=container-foo-%i \
+	--cidfile=%t/%n.ctr-id \
+	--cgroups=no-conmon \
+	--rm \
+	--sdnotify=conmon \
+	-d awesome-image:latest
```

Comment:
Just curious why you didn't have the argument(s) on their own line for each of these ala:
```suggestion
	-d \
	awesome-image:latest
```

Reply:
The original patch did this, and it was felt that this did not look good, it broke the line up into two many lines. @Luap99 and I agreed this method looked best.